target/linux/kirkwood/base-files/etc/init.d/hwmon_fancontrol

```diff
@@ -11,6 +11,10 @@ boot() {
 		path_to_hwmon='/sys/class/hwmon/hwmon0'
 		echo 2 > "$path_to_hwmon/pwm1_enable" # fan is on pwm1
 		;;
+	netgear,readynas-duo-v2)
+		path_to_hwmon='/sys/class/hwmon/hwmon0'
+		echo 1200 > "$path_to_hwmon/fan1_target" #set target rpm
```

Comment:
(I cannot add a comment on a line that is not in the diff, so I'll write this here)

> `# configuring (lm85/lm63) onboard temp/fan controller to run the fan on its own`

Maybe we can add "g762" to `(lm85/lm63)`, or remove it entirely.

And could you add a space after `#`?